checklists/lz_checklist.en.json

```diff
@@ -796,7 +796,7 @@
       "guid": "6138a720-0f1c-4e16-bd30-1d6e872e52e3",
       "severity": "Medium",
       "training": "https://docs.microsoft.com/learn/paths/secure-application-delivery/",
-      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups#management-groups-in-the-azure-landing-zone-accelerator"
+      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups"

```

Comment:
Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups#management-groups-in-the-azure-landing-zone-accelerator

Reply:
Should be fixed now - thank you for spotting this.